setup.py

```diff
@@ -13,9 +13,10 @@
     include_package_data=True,
     packages=find_packages(),
     install_requires=[
+        'oauthlib==3.1.0'
```

Comment:
What's this module for? I don't see it being used anywhere.

Reply:
jire requires it, but has some fail in package definition. If I installed it I have conflict with versions again..